recipes/netcdf/all/conanfile.py

```diff
@@ -121,22 +106,13 @@ def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "netCDF")
         self.cpp_info.set_property("cmake_target_name", "netCDF::netcdf")
         self.cpp_info.set_property("pkg_config_name", "netcdf")
-        # TODO: back to global scope in conan v2 once cmake_find_package_* generators removed
-        self.cpp_info.components["libnetcdf"].libs = ["netcdf"]
+        self.cpp_info.libs = ["netcdf"]
         if self._with_hdf5:
-            self.cpp_info.components["libnetcdf"].requires.append("hdf5::hdf5")
+            self.cpp_info.requires.append("hdf5::hdf5")
         if self.options.dap or self.options.byterange:
-            self.cpp_info.components["libnetcdf"].requires.append("libcurl::libcurl")
+            self.cpp_info.requires.append("libcurl::libcurl")
         if self.settings.os in ["Linux", "FreeBSD"]:
-            self.cpp_info.components["libnetcdf"].system_libs = ["dl", "m"]
```

Comment:
these ones we are keeping out of caution, as they part of the interface of the recipe and can be referenced outside
so this probably needs to stay :(